Cannot run docker tests

I'm on OSX and attempting to run the docker tests:

    > mkdir release
    > cd release
    > cmake ..
    > cpack -G TGZ
    > cd ..
    > docker build -t mps .

This fails with:

    Step 12/13 : RUN ln -s /usr/bin/cmake3 /usr/bin/cmake; cmake ..; make
     ---> Running in 3434eb7992c9
    CMake Error: The current CMakeCache.txt directory /app/release/CMakeCache.txt is different than the directory /Users/dave/workspace/mozilla-pipeline-schemas/release where CMakeCache.txt was created. This may result in binaries being created in the wrong place. If you are not sure, reedit the CMakeCache.txt
    CMake Error: The source "/app/CMakeLists.txt" does not match the source "/Users/dave/workspace/mozilla-pipeline-schemas/CMakeLists.txt" used to generate cache.  Re-run cmake with a different source directory.
    make: /usr/local/Cellar/cmake/3.11.2/bin/cmake: Command not found
    make: *** [cmake_check_build_system] Error 127
    The command '/bin/sh -c ln -s /usr/bin/cmake3 /usr/bin/cmake; cmake ..; make' returned a non-zero code: 2

--- a/.dockerignore
+++ b/.dockerignore
@@ -1,0 +1,1 @@
+release/

